packaging_automation/package_cloud_statistics_collector.py

```diff
@@ -167,5 +174,6 @@ def stat_records_exists(download_date: date, package_full_name: str, session) ->
 
     fetch_and_save_package_cloud_stats(db_parameters, arguments.package_cloud_api_token,
                                        PackageCloudOrganizations[arguments.organization],
-                                       PackageCloudRepos[arguments.repo_name], arguments.parallel_count,
+                                       PackageCloudRepos(arguments.repo_name),
+                                       arguments.parallel_count,
```

Comment:
What is the change here?

Reply:
I was using keys to get the enum value. I changed it to use value of the enum e.g. before community_nightlies was used now community-nightlies can be used